### ISSUE TITLE
What is the best way to communicate changes in axioms to a user

### ISSUE THREAD

**Issue by matentzn** [INCATools/boomer#279]
Consider this:

- the user provides mappings in a ptable of the "exact" variety
- then boomer applies these, but changes some of them to "subclassof"

We should communicate this clearly to the user. The question is: should this be implemented as a mappingset diff? I.e. Boomer exports and sssom mapping table alongside the owlfile and we show a sssom diff to the user between the input mappings and the output? Or should boomer say something in a nicely formatted markdown?

**Comment by cmungall** [comment]
boomer will never change axioms

if a user supplies:

|sub|sup|eq|sib|
|---|---|---|---|
|0.1|0|0.9|0|

and boomer selects `sub` as globally the most likely then this is communicated in a few ways:

 - the markdown will show a low axiom-level probability (0.1) and will not say "most probable"
 - the lines in the graphviz will be thin unlike fat likely axioms

but boomer will never select sub if the prior is zero, it never 'changes' anything, it just selects the best possible consistent world (of the ones it was able to explore)

**Comment by matentzn** [comment]
Yes, something in this direction is fine, but I still believe that a table with all the lowest priority "adoptions" would be important - I think you mentioned this before.
